bitnami/harbor/Chart.yaml

```diff
@@ -34,4 +34,4 @@ sources:
   - https://github.com/bitnami/bitnami-docker-harbor-registry
   - https://github.com/bitnami/bitnami-docker-harbor-registryctl
   - https://goharbor.io/
-version: 9.2.2
+version: 9.2.3
```

Comment:
Since there are new parameters you should bump the minor version

Reply:
Done